index.js

```diff
@@ -248,3 +250,37 @@ function generateImage (opt, callback) {
     callback(null, container);
   });
 }
+
+function RoundAllValue (obj, decimal = 0) {
+  for (var key in obj) {
+    if (obj.hasOwnProperty(key)) {
+      if (obj[key] !== null) {
+        if (typeof(obj[key]) == "object") {
+          RoundAllValue (obj[key], decimal);
+        } else {
+          if(isNumeric(obj[key])) {
+            var oldNumber = obj[key];
+            obj[key] = roundNumber(obj[key], decimal);
+          }
+        }
+      }
+    }
+  }
+}
+
+function isNumeric (n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
```

Comment:
I believe `global.isNaN` coerces its argument to a number anyways, so the parseFloat call is unnecessary. Additionally, `isFinite(NaN)` will return false, so I believe use of this function can just be replaced with `isFinite`.